src/Coverage/Transformer.hs

```diff
@@ -54,14 +55,29 @@ transform modul =
                 |> List.map (\(AST.UppercaseIdentifier s) -> s)
                 |> intercalate "."
 
+        dropEnd :: Int -> [a] -> [a]
+        dropEnd i xs = reverse . drop i . reverse $ xs
+
+        fileRootDirPath :: FilePath
+        fileRootDirPath =
+            filePath
+                |> splitDirectories
+                |> drop 2 -- .coverage/instrumented/
+                |> dropEnd (length moduleName_)
```

Comment:
why don't you use take instead?

Reply:
I guess I could, calculating the amount from this and from the original length of split filePath...

Reply:
yes. I would probably do that because it eliminates need for that extra helper function and even saves one reverse. I think it would be great if elm-coverage would support multiple src directories.